src/System.Runtime.Extensions/ref/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.20"
+    "System.Runtime": "4.1.0"
   },
   "frameworks": {
     "netstandard1.7": {
```

Comment:
This will generate a warning when you build the netcoreapp1.1 configuration.  You can also delete the imports, they're no longer needed for the package version you are referencing.